pkg/registry/lock/registry.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
I don't think it's documented anywhere.

Update: I renamed all "rest.go" to "strategy.go" to be more intuitive. Also note, that registry.go is usually not needed.